app/views/catalog/_form_basic_info.html.haml

```diff
@@ -6,65 +6,72 @@
     = render(:partial=>'layouts/ae_tree_select')
 
   %h3=_('Basic Info')
-  %table.style1
-    %tr
-      %td.key=_('Name / Description')
-      %td
-        = text_field_tag("name",
+  %form.form-horizontal
+    .form-group
+      %label.col-sm-2
+        =_('Name / Description')
+      .col-sm-10
+        %p.form-control-static
```

Comment:
I think `form-control-static` is meant only for elements of the form that are not inputs or selects. For inputs and selects, you want form-control, directly on the input - so `= text_field_tag( ... :class => 'form-control')`